doc/functions/hysteresis.rst

```diff
@@ -0,0 +1,15 @@
+Hysteresis
```

Comment:
Just a note on adding it to the core. I think it's a bit too specific to do that but I'm currently working on a plugin with useful extra functions I'd like to include it in. Where I'd include small useful things nobody can be bothered with elsewhere but are still good or replaces some avisynth functionality. I'm currently working on a function to average frames to finally evict temporalsoften from all scripts as well.